src/panels/lovelace/cards/hui-gauge-card.ts

```diff
@@ -123,33 +122,33 @@ class HuiGaugeCard extends LitElement implements LovelaceCard {
       `;
     }
 
+    const sliderBarColor = this._computeSeverity(state);
+
     return html`
       <ha-card
-        @click="${this._handleClick}"
+        @click=${this._handleClick}
         tabindex="0"
         style=${styleMap({
           "--base-unit": this._baseUnit,
```

Comment:
What does this do and why do we need it? The round slider can just scale with the parent?

Reply:
Is it used just for the `path-width`?

Reply:
Path width, fontsize, slider width, distance between slider and text

Reply:
I would really love to be able to drop the base unit and do things based on percentage where possible